utilities/src/parallel.rs

```diff
@@ -76,18 +76,18 @@ pub fn max_available_threads() -> usize {
 }
 
 #[inline(always)]
-#[cfg(feature = "parallel")]
+#[cfg(all(feature = "parallel", not(feature = "web")))]
 pub fn execute_with_max_available_threads<T: Sync + Send>(f: impl FnOnce() -> T + Send) -> T {
     execute_with_threads(f, max_available_threads())
 }
 
-#[cfg(not(feature = "parallel"))]
+#[cfg(any(not(feature = "parallel"), feature = "web"))]
 #[inline(always)]
 pub fn execute_with_max_available_threads<T>(f: impl FnOnce() -> T + Send) -> T {
     f()
 }
 
-#[cfg(feature = "parallel")]
+#[cfg(all(feature = "parallel", not(feature = "web")))]
 #[inline(always)]
 fn execute_with_threads<T: Sync + Send>(f: impl FnOnce() -> T + Send, num_threads: usize) -> T {
     let pool = rayon::ThreadPoolBuilder::new().num_threads(num_threads).build().unwrap();
```

Comment:
Motivation for this change:

With the updates to the aleo wasm here: https://github.com/demox-labs/aleo/pull/6/files
a global thread pool is setup with WebWorkers as the receivers. Due to this, the functions `f` will run in that global, worker-backed threadpool.
Attempting to create a new threadpool from within the WebWorkers causes a panic.